### PR TITLE
TAO-7816 Added unique entity exception

### PR DESCRIPTION
**Pull request by gyszucs**
- Throw exceptions instead of catching them to let the main application to handle it.
- Added specific exception if the `sourcedId` is not unique in any CSV.

### CODE REVIEW

File: src/Schema/UniqueEntityException.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace oat\OneRoster\Schema;
+
+class UniqueEntityException extends \RuntimeException
```
Comment (chilly86) [inline src/Schema/UniqueEntityException.php]: please name it NotUniqueEntity because that's what it means

File: composer.json [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
+            "oat\\OneRoster\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "oat\\OneRoster\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "php": "^7.2",
```
Comment (chilly86) [inline composer.json:28]: do we want to be so strict with the php version?

**Review by chilly86 — CHANGES_REQUESTED** [review]
Seems okay. Tests are running.

File: src/Schema/UniqueEntityException.php (hunk 1 above)
Comment (gyszucs) [reply src/Schema/UniqueEntityException.php]: done

**Pull request merged**